content/blog/2023-10-05-october-Prism-launch/contents.lr

```diff
@@ -0,0 +1,21 @@
+title: Prism: The New Feinberg Repository for Global Dissemination of Research
+---
+pub_date: 2023-10-10
```

Comment:
minor: update the date to when publication will happen.

Reply:
Agree!